configuration/configuration_storage.go

```diff
@@ -85,7 +84,7 @@ type configTypeReload struct {
 	ReloadDelay     *int    `yaml:"reload_delay,omitempty" hcl:"reload_delay,omitempty"`
 	ReloadCmd       *string `yaml:"reload_cmd,omitempty" hcl:"reload_cmd,omitempty"`
 	RestartCmd      *string `yaml:"restart_cmd,omitempty" hcl:"restart_cmd,omitempty"`
-	StatusCmd      	*string `yaml:"status_cmd,omitempty" hcl:"status_cmd,omitempty"`
+	StatusCmd       *string `yaml:"status_cmd,omitempty" hcl:"status_cmd,omitempty"`
```

Comment:
Please remove whitespace here.

Reply:
Just checked, there's no whitespace here. I think this diff is just showing that L88 -> L87. I ran `go fmt` on this file and it seems fine.